mod-orders-storage/examples/alert_collection.sample

```diff
@@ -4,8 +4,8 @@
           {
             "alert": "Receipt overdue",
             "id": "9a665b22-9fe5-4c95-b4ee-837a5433c95d",
-            "po_line_id": "8c778aee-97fa-4586-b131-3ea588a728e2"
+            "poLineId": "8c778aee-97fa-4586-b131-3ea588a728e2"
```

Comment:
Schema `alert.json` do not have `poLineId` field anymore. So it should be deleted from the sample
```suggestion
```